sdk/mongocluster/arm-mongocluster/test/public/node/privateEndpointConnections.spec.ts

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
Is there an issue in the SDK repo to figure out what is the root cause for browser failure for it ?